routes/teams.js

```diff
@@ -35,7 +35,7 @@ router.get('/quiz', function(req, res) {
 
   if (req.query.trending){
     if (type){ // leaders or goats
-      db.collection('teams').findOne( { _id : rb_team_id}, function (err, item){
+      db.collection('teams').findOne( { team_id : rb_team_id}, function (err, item){
```

Comment:
Im partial to B too. The main reason it is slow is because when we were querying by _id. Mongo automatically indexes that field. Let's put unique indexes on the team_id fields in every collection. We should do it via script so we have record off it.